### PR TITLE
Closes #1327

### LINKED ISSUE #1327: False positive for WPS125 and attributes
# Bug report

## What's wrong

```
  76:1     WPS125 Found builtin shadowing: __doc__
  _coalesce_result.__doc__ = __doc__
  ^
```

It is not technically builtin shadowing.

### PR DESCRIPTION
**Pull request by sobolevn**

### CODE REVIEW

File: tests/test_visitors/conftest.py [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
 from wemake_python_styleguide.violations.base import (
     ASTViolation,
+    BaseViolation,
     TokenizeViolation,
 )
 from wemake_python_styleguide.visitors.base import BaseVisitor
 
+_IgnoredTypes = Union[
+    Type[BaseViolation],
+    Tuple[BaseViolation, ...],
```
Comment (skarzi) [inline tests/test_visitors/conftest.py:14]: Probably it should be: 

```python
Tuple[Type[BaseViolation], ...],
```

File: tests/test_visitors/test_ast/test_naming/test_naming_rules/test_reserved_argument.py [hunk 2]
```diff
@@ -23,47 +27,28 @@ def method({0}, different):
...
     default_options,
-    code,
+    naming_template,
     mode,
     argument,
 ):
     """Ensures that special names for arguments are restricted."""
-    tree = parse_ast_tree(mode(code.format(argument)))
+    with suppress(SyntaxError):
+        # We use `suppress` here because some fixutres
```
Comment (skarzi) [inline tests/test_visitors/test_ast/test_naming/test_naming_rules/test_reserved_argument.py:43]: * fixtures

**Comment by sobolevn** [comment]
@skarzi thanks a lot, fixed! 

Sorry for rushing with the merge, but I want to release `0.15` ASAP after #1847

**Comment by skarzi** [comment]
You're welcome!
Sure I understand, great work as always :+1:

**Pull request merged**